src/runtime/cli/config/configuration-qemu.toml.in

```diff
@@ -314,10 +321,10 @@ valid_entropy_sources = @DEFVALIDENTROPYSOURCES@
 # the OCI spec passed to the runtime.
 #
 # You can create a rootfs with hooks by customizing the osbuilder scripts:
-# https://github.com/kata-containers/kata-containers/tree/main/tools/osbuilder
+# https://github.com/kata-containers/osbuilder
 #
 # Hooks must be stored in a subdirectory of guest_hook_path according to their
-# hook type, i.e. "guest_hook_path/{prestart,poststart,poststop}".
```

Comment:
The same is true for this line.
Cloud you please put it back?

Reply:
Thank you @ManaSugi , I've updated the code according to your comments.